client/app/bundles/course/assessment/utils/recorderHelper.js

```diff
@@ -0,0 +1,73 @@
+import Recorder from '../../../../../vendor/recorderjs';
```

Comment:
Any issues if we define the `client/vendor` folder as one of the defaults paths to search in - any issues @allenwq ?

Reply:
I tried to added it in the ```webpack.config```. It looks ok, but travis will throw error. Not sure if I did it wrongly.

Reply:
Did you add supplement the `cache` in either `package.json` or the travis configuration?

Reply:
No, I only added in the ```alias``` in the webpack.config